packages/moda-sdk/src/profile/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './createProfile';
+export * from './createProfileFor';
```

Comment:
we should export all public functions here.

We should also add functions for every public method in the contract. e.g. `accountUri`, `ownerOf`, etc